src/electron/ipc-api/autoUpdate.ts

```diff
@@ -24,11 +25,14 @@ export default (params: { mainWindow: BrowserWindow; settings: any }) => {
             autoUpdater.checkForUpdates();
           } else if (args.action === 'install') {
             debug('installing update');
-            app.removeAllListeners('window-all-closed');
-            params.mainWindow.removeAllListeners('close');
+
+            appEvents.emit('install-update');
+
+            const openedWindows = BrowserWindow.getAllWindows();
+            for (const window of openedWindows)  window.close();
+
             autoUpdater.quitAndInstall();
             // TODO: based on https://github.com/electron-userland/electron-builder/issues/6058#issuecomment-1130344017 (not yet tested since we don't have signed builds yet for macos)
-            app.exit();
```

Comment:
we need to test whether this is needed or not - with the latest electron version we use. Or have you tested this after rebasing? If so, then the line deletion is valid (and if so, please delete the above `TODO` line as well)